ingress/controllers/nginx/README.md

```diff
@@ -149,6 +150,23 @@ By default the controller redirects (301) to HTTPS if there is a TLS Ingress rul
 To disable this behavior use `hsts=false` in the NGINX config map.
 
 
+### Automated Certificate Management with Kube-Lego
+
+[Kube-Lego] automatically requests missing certificates or expired from
+[Let's Encrypt] by monitoring ingress resources and its referenced secrets. To
+enable this for an ingress resource you have to add an annotation:
+
+```
+kubectl annotate ing ingress-demo kubernetes.io/tls-acme="true"
```

Comment:
The way it's currently written, I believe kube-lego will create an ingress for itself that exposes /well-known (just by looking at code, not tried). 

This will only work with Ingress controller that serve all Ingresses from the same ingress backend. On cloudproviders, each ingress is a different public vip, so if I provision example.com -> my ingress vip, and kube-lego creates another ingress for /well-know, it gets another IP (so example.com/well-known won't resolve).

We should just document that people should create a rule like:
```
      - backend:
          serviceName: kube-lego
          servicePort: 8080
        path: /.well-known/acme-challenge/*
```

then kube-lego just needs to talk acme. 

Creating a new ingress is a nice trick, for it to work we need ip/hostname claims.